src/components/ImageView/index.native.js

```diff
@@ -104,23 +104,21 @@ class ImageView extends PureComponent {
      */
     configureImageZoom({nativeEvent}) {
         // Wait till animations are over to prevent stutter in navigation animation
```

Comment:
I think this is now outdated right, it was the reason for why were using `InteractionManager`?